shared-services/outputs.tf

```
@@ -9,6 +9,6 @@ output "provisionaccount_role" {
 }
 
 output "ssmsession_role" {
-  value       = aws_iam_role.ssmsession_role
+  value       = module.session_manager.ssm_session_role
   description = "The IAM role that allows creation of SSM SessionManager sessions to any EC2 instance in this account."
```

So this is where it all started.....
```suggestion
  description = "The IAM role that allows creation of SSM Session Manager sessions to any EC2 instance in this account."
```

---

Please see commit 9264e5f.